src/components/cc-invoice-list/cc-invoice-list.js

```diff
@@ -48,14 +57,18 @@ export class CcInvoiceList extends LitElement {
     /** @type {InvoiceListState} Sets the invoices state. */
     this.state = { type: 'loading' };
 
-    /** @type {number|null} */
+    /** @type {string|null} */
```

Comment:
**🔨 issue:** If I understand correctly, should be updated as well in the `properties()` getter where it is still typed as a number (line 50).

```js
  static get properties () {
    return {
      state: { type: Object },
      _yearFilter: { type: Number, state: true },
    };
  }
```